File: packages/engine/tests/units/state/nullable_fixed_size_list/src/behaviors/empty.py
def behavior(state, context):
    """
    Regression test with at least three agents, where one agent has a fixed-size list as field, but one agent before and
    one agent after does not have this field.

    Empty behavior (we need a behavior to execute)
    """
    state.list_is_null = state.list is None

File: packages/engine/tests/units/state/nullable_fixed_size_list/src/behaviors/empty.js
/**
 * Regression test with at least three agents, where one agent has a fixed-size list as field, but one agent before and
 * one agent after does not have this field.
 *
 * Empty behavior (we need a behavior to execute)
 */
const behavior = (state, context) => {
  state.list_is_null = state.list === null;
};
